lib/class-wp-rest-url-details-controller.php

```diff
@@ -66,8 +66,26 @@ public function get_item_schema() {
 			'title'      => 'url-details',
 			'type'       => 'object',
 			'properties' => array(
-				'title' => array(
-					'description' => __( 'The contents of the <title> tag from the URL.', 'gutenberg' ),
+				'title'       => array(
+					'description' => __( 'The contents of the <title> element from the URL.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'icon'        => array(
+					'description' => __( 'The favicon image link of the <link rel="icon"> element from the URL.', 'gutenberg' ),
```

Comment:
I think this can get a `format` of `uri` right?

Reply:
And same for `image`.

Reply:
You're right @TimothyBJacobs. Updated both in commit https://github.com/WordPress/gutenberg/pull/31763/commits/b013a82031c326bffe0b2676d2f21798b0eb2cfb

Reply:
Sorry, to clarify, the `type` is still `string` but there should be a `format` keyword with `uri`.

Reply:
Doh, of course. Sorry about that. Reverted and added `format`  in commit https://github.com/WordPress/gutenberg/pull/31763/commits/0097a7407c4cf3dfec7d94f476b0a6c36844714c